src/@ionic-native/plugins/preview-any-file/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
+/**
+ * @name Preview Any File
+ * @description
+ * This plugin does something
```

Comment:
Please add a description

Reply:
please check the new commit 
```suggestion
 *  * Whatever the file is PDF document, Word document, Excel, office document,zip archive file, image, text, html or anything else, you can perform a preview by this cordova Plugin to preview any file in native mode by providing the local or external URL.

```